content/gamenews

```diff
@@ -1,4 +1,4 @@
-2015-08-30 03-00-00:
-	Title: Playtest 20150830
-	Author: Pavel Penev
-	Content: We are happy to announce that another playtest on the way to our next release is now available!\n\nThis one focused on fixing bugs and polishing the existing features.\n\nHave a look at http://openra.net for details!\n\nAs always with our playtests, we invite you to report any issues you encounter at http://bugs.openra.net to ensure the final release is as bug-free as possible. Thanks, and enjoy!
+2015-09-12 12-00-00:
+	Title: Release 20150919
+	Author: Paul Chote
+	Content: LANDCOM 16 HQS.\nTOP SECRET\nTO: ALL FIELD COMMANDERS\n\nAN INTERCEPTED COMMUNIQUE INDICATES THE\nIMMEDIATE AVAILABILITY OF A NEW SEPTEMBER\n2015 OPENRA RELEASE.  SUBSEQUENT INTERROGATION OF THE MESSAGE COURIER REVEALED THAT 31 AUTHORS CONTRIBUTED\nMORE THAN 1150 CHANGES TOWARDS THEIR\nWAR EFFORT IN THE LAST THREE MONTHS.  A LARGE COLLECTION OF BUG FIXES AND OTHER\nIMPROVEMENTS ARE SUSPECTED.\n\nYOUR ORDERS ARE TO PROCEED DIRECTLY\nTO THE OPENRA WEBSITE AND DOWNLOAD THE NEW RELEASE.  RENDEZVOUS WITH OTHER COMMANDERS IN-GAME AND DESTROY ALL ENEMY OPPOSITION.\n\nTRANSMISSION ENDS.
```

Comment:
The line breaks are off:
![screenshot from 2015-09-19 12 58 55](https://cloud.githubusercontent.com/assets/4331210/9975921/8d474248-5ece-11e5-8bef-ae8cd7ec43bc.png)